hnn_core/network.py

```diff
@@ -310,10 +312,62 @@ def _instantiate_feeds(self, n_trials=1):
                                 params=p_unique[src_type],
                                 gid=gid)
                         )
-                event_times_per_source.update({src_type: event_times})
+                feed_times[src_type].append(event_times)
+
+        self.feed_times = feed_times
+
+        # Add tonic inputs from param files
+        self.feed_times['tonic'] = dict()
+        for cell_type in ['L2Basket', 'L5Basket', 'L2Pyr', 'L5Pyr']:
+            is_tonic_present = [f'Itonic_{p}_{cell_type}_soma' in self.params
+                                for p in ['A', 't0', 'T']]
+            if any(is_tonic_present):
+                if not all(is_tonic_present):
+                    raise ValueError(f'Tonic input must have the amplitude, '
+                                     f'start time and end time specified. One '
+                                     f'or more parameter may be missing for '
+                                     f'cell type {cell_type}')
+                self.add_tonic_input(
+                    cell_type=cell_type,
+                    amplitude=self.params[f'Itonic_A_{cell_type}_soma'],
+                    t0=self.params[f'Itonic_t0_{cell_type}_soma'],
+                    T=self.params[f'Itonic_T_{cell_type}_soma']
+                )
+
+    def add_tonic_input(self, cell_type, amplitude, t0, T):
```

Comment:
This is nitpicky, but I find it confusing to have an `add_tonic_input()` method for both the `Network` and `_Cell` objects. Unless we're planning on immediately refactoring more of this in the next PR, would it be possible to change this one to `Network.add_tonic_inputs()` (note the plural usage) or something of the sort?

Reply:
Can I take this in #221 too? It'll be under 'attach_bias' under Network.